config/config/common-config.yaml

```diff
@@ -120,9 +131,7 @@ falco:
   ## additional falco rules
   ## ref: https://falco.org/docs/rules/
   customRules:
-    non-sudo-setuid.yaml: |-
-      - rule: Non sudo setuid
-        enabled: false
```

Comment:
Just to make this more clear I think I'd want an example here in a comment. Something like this, but feel free to update it:
```
customRules: {}
  # my-rules-example.yaml : |-
  # - macro: <name of macro to add/change> # possible to use rules and lists as well
  #   condition: <add condition here>
  #   ...
```